src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -325,10 +325,10 @@ public async Task TryInjectBrowserScriptAsync(string contentType, string request
             if (rumBytes == null)
             {
                 transaction.LogFinest("Skipping RUM Injection: No script was available.");
-                await baseStream.WriteAsync(buffer, 0, buffer.Length);
+                await baseStream.WriteAsync(buffer, 0, buffer.Length).ConfigureAwait(false);
```

Comment:
Are we sure that this is safe? I don't know if there are scenarios that working with this stream will require dealing with the synchronization context.

Reply:
As our instrumentation is essentially "library" code, it is recommended to use `.ConfigureAwait(false)` on all `await`ed async method calls. I can't imagine any scenario where this would be unsafe.